lib/std/sums.nim

```diff
@@ -53,6 +53,136 @@ func sumPairs*[T](x: openArray[T]): T =
   let n = len(x)
   if n == 0: T(0) else: sumPairwise(x, 0, n)
 
+func fastTwoSum*[T : SomeFloat](a, b: T): (T, T) =
```

Comment:
How about a simple `assert`

Reply:
> Btw noob question, is the `fastTwoSum`, `twoSum`, `shewchuckSum_add` and `shewchuckSum_total` really useful for users? Because it seems to me they should be private functions.

TwoSum, yes
fastTwoSum, maybe (on ordered list for example)
shewchuckSum_add / shewchuckSum_total, don't know. shewchuckSum should be sufficent.

Reply:
if it's going to be private, there is no need for assert